File: HRSystem.DTO/UserListDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace HRSystem.DTO
{
    public class UserListDto
    {
        public int Id { get; set; }
        public string Username { get; set; }
        public string Address { get; set; }
        public string Email { get; set; }
        public string Mobile { get; set; }
        public string BirthDate { get; set; }
        public int? ManagerId { get; set; }
        public int typeId { get; set; }
        public string Manager { get; set; }

    }
}

File: HRSystem.API_SPA/ClientApp/src/app/Models/UserListDto.ts
export class UserListDto {
    Id :number;
    Username:string;
    Address :string;
    Email :string;
    Mobile :string;
    BirthDate:string;
    ManagerId :number;
    typeId :number;
    Manager :string

    constructor(Id,
        Username,
        Address ,
        Email,
        Mobile,
        BirthDate,
        ManagerId ,
        typeId,
        Manager
    ){
        this.Id = Id;
        this.Username = Username;
        this.Address = Address;
        this.Email = Email;
        this.Mobile = Mobile;
        this.BirthDate = BirthDate;
        this.ManagerId = ManagerId;
        this.typeId = typeId;
        this.Manager = Manager;
    }
}
